### ISSUE TITLE
Feature Request: Give config file and credentials

### ISSUE THREAD

**Issue by thethinker990** [Jamie-/openvpn-api#29]
Hi,

in the CLI-version of openvpn it is possible to pass a config file and credentials:
```batch
openvpn --config ".\server.ovpn" --auth-user-pass ".\credentials" --auth-nocache
```

In `credentials` (line 1+2)
```
username
password
```

Is it possible to put this into the API?

**Comment by Avnsx** [comment]
Yeah definetly would need this, I was looking forward to being able to do it like this to connect to ipvanish servers, repo is kinda missing the point without this

**Comment by MohammadRaziei** [comment]
I need it plus I want to support that enter OTP automatically using secret key
